aten/src/ATen/cpu/sleef/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+IF(MSVC)
+  option(BUILD_SHARED_LIBS "Build shared libs" ON)
+ELSE(MSVC)
```

Comment:
You don't have to fix this, but FYI modern cmake style is to not repeat the conditional in else/elseif/endif. So:

```
if(MSVC)
  ...
else()
  ...
endif()
```